rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -227,7 +227,44 @@ void debugger_frame::keyPressEvent(QKeyEvent* event)
 	const auto cpu = this->cpu.lock();
 	int i = m_debugger_list->currentRow();
 
-	if (!isActiveWindow() || !cpu)
+	if (!isActiveWindow())
```

Comment:
This should be at the beginning of the function, and the cpu and currentRow stuff after your new switch

Reply:
Done.